README.rst

```diff
@@ -0,0 +1,109 @@
+steenroder: A Python package for the computation of persistence Steenrod barcodes
+=================================================================================
+
+The widespread use in applied topology of the barcode of filtered
+cellular complexes rests on a balance between discriminatory power and
+computability. It has long been envision that the strength of this
+invariant could be increase using cohomology operations. This package
+computes the recently defined *Sq*\ \ *k*\ -barcodes which have been
+shown to effectively increase the discriminatory power of barcodes on
+real-world data.
+
+For a complete presentation of these invariants please consult
+`Persistence Steenrod modules <https://arxiv.org/abs/1812.05031>`__ by
+U. Lupo, A. Medina-Mardones and G. Tauzin.
+
+License
+-------
+
+``steenroder`` is distributed under the `MIT
+license <https://github.com/Steenroder/steenroder/LICENSE>`__.
+
+Documentation
+-------------
+
+Please visit https://steenroder.github.io/steenroder/ and navigate to
+the version you are interested in.
+
+Installation
+------------
+
+Dependencies
+~~~~~~~~~~~~
+
+The latest stable version of ``steenroder`` requires:
+
+-  python (>= 3.8)
```

Comment:
Can we not support 3.7?

Reply:
We can!

Reply:
Actually, we can't. The CI python 3.7 fails with:

```
 steenroder/core.py:44: in <module>
    @lru_cache
/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/functools.py:490: in lru_cache
    raise TypeError('Expected maxsize to be an integer or None')
E   TypeError: Expected maxsize to be an integer or None
```